app/src/components/Registration/Checkbox.tsx

```diff
@@ -8,19 +8,21 @@ import {
     Checkbox,
 } from '@material-ui/core'
 
-const CheckboxInput = (props: {
+interface Props {
     onChange: (event: object) => void
-    className: string
     label: string
     options: Array<string>
-    checked: boolean
     helperText: string
-}) => {
-    const { onChange, className, label, options, checked, helperText } = props
+}
+
+//checkbox for multi-select
+const CheckboxInput = (props: Props) => {
+    const { onChange, label, options, helperText } = props
+    let checked: boolean
 
     return (
         <>
-            <form className={className}>
+            <form>
```

Comment:
Epic Components Arshia!
We'll have to decide if we need <form> elements when we tackle validation. Since they can't be nested, it would be tricky to conditionally include them based on whether a parent component has already included one.

Reply:
great point @martyboggs! those can definitely be removed. nice catch.